modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesRecoveryTask.java

```diff
@@ -169,23 +163,19 @@ private void reloadQueuesFromDB() throws AndesException {
 
 			queuesStored.removeAll(queueList);
 			for (AndesQueue queue : queuesStored) {
-				for (QueueListener listener : queueListeners) {
-					log.warn("Recovering node. Adding queue " + queue.toString());
-					/**
-					 * Ignoring MQTT queues when recovering as they are already stored in the database. 
-					 */
-					if (queue.getProtocolType() != ProtocolType.MQTT) {
-						listener.handleClusterQueuesChanged(queue, QueueListener.QueueEvent.ADDED);
-					}
+				log.warn("Recovering node. Adding queue " + queue.toString());
+				/**
+				 * Ignoring MQTT queues when recovering as they are already stored in the database.
+				 */
+				if (queue.getProtocolType() != ProtocolType.MQTT) {
```

Comment:
I guess @akalankapagoda has already removed this in [c5](https://github.com/wso2/andes/blame/1f8f29684f8919ba0a646b4e319eb05599108c32/modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesRecoveryTask.java)